Keyboards/KeyboardsBase/KeyboardViewController.swift

```diff
@@ -406,7 +406,9 @@ class KeyboardViewController: UIInputViewController {
           var i = 0
           if completionOptions.count <= 3 {
             while i < completionOptions.count {
-              if shiftButtonState == .caps {
+              if shiftButtonState == .shift {
```

Comment:
Interesting, we were missing shift?? Great catch :)

Reply:
Yup, suggestions were not handled properly for capitalised first-letter but it will be history after the merge.